config/config.go

```diff
@@ -307,9 +307,10 @@ func ValidateEmail(email string) bool {
 }
 
 func ValidateEndpoints(daemonEndpoint string, passthroughEndpoint string) error {
+
 	passthroughURL, err := url.Parse(passthroughEndpoint)
-	if err != nil {
-		return errors.New("passthrough endpoint must be a valid URL")
+	if err != nil || passthroughURL.Host == "" {
+		return errors.New("passthrough_endpoint must be a valid URL")
```

Comment:
Lets make the error message descriptive. Something like- You need to set the passthrough_endpoint variable in the daemon config to the endpoint of your AI service"

Reply:
done